.gitignore

```diff
@@ -2,4 +2,4 @@
 composer.lock
 build
 coverage.xml
-.phpunit.result.cache
+.phpunit.result.cache
```

Comment:
This is removing a final new line, which is defined in the editorconfig file. It might be best to just check this file out of the changes or install the editorconfig plugin for your editor to ensure the rules are respected
https://github.com/tymondesigns/jwt-auth/blob/develop/.editorconfig#L6

Reply:
Had to merge with the upstream and it encountered a conflict, so I copied from the upstream's file. We both have the same content. Should I have to put an extra newline at the end of this file?

Reply:
i'd rather wget your gitignore, and will fix it then.

Reply:
Updated the `.gitignore`. Please review.